.github/pull-request-template.md

```diff
@@ -0,0 +1,21 @@
+Fixes #
+
+## Proposed Changes
+
+*
+*
+*
+
+**Release Note**
+
+<!-- Enter your extended release note in the below block. If the PR requires
+additional action from users switching to the new release, include the string
+"action required". If no release note is required, write "NONE". -->
+
+```release-note
+
+```
+
+## Steps to test
```

Comment:
I'm not sure about this. Normally we should know how to test the changes if proper unit and integration tests were added. This section makes it seem like ad-hoc testing is the standard. I would consider not having this section and also not mention anything about testing requirements, which would allow us to accept PRs at our own discretion.

Reply:
Okay, I updated to remove this section but also added more instructions to format the changelog so we can parse it automatically.